backend/server/models/question.js

```diff
@@ -0,0 +1,43 @@
+
+module.exports = (sequelize, DataTypes) => {
+  const Question = sequelize.define('Question', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: DataTypes.INTEGER,
+    },
+    text: {
+      allowNull: false,
+      type: DataTypes.TEXT,
+    },
+    initialImage: {
+      allowNull: false,
+      type: DataTypes.TEXT,
+    },
+    finalImage: {
+      allowNull: false,
+      type: DataTypes.TEXT,
+    },
+    initialSound: {
+      allowNull: false,
+      type: DataTypes.TEXT,
+    },
+    finalSound: {
+      allowNull: false,
+      type: DataTypes.TEXT,
+    },
+  }, {});
+
+  Question.associate = (models) => {
+    Question.belongsTo(models.User, {
+
+      foreignKey: 'user_id',
+      as: 'user',
+      onDelete: 'CASCADE',
+    });
+    Question.belongsToMany(models.Questionnaire, { through: 'QuestionQuestionnaire' });
```

Comment:
Each question should belong to an experiment, not a user. Please, fix this, @maverick2000. Thanks.